components/linear_app/linear_app.app.mjs

```diff
@@ -171,6 +171,19 @@ export default {
     async listIssueLabels(variables = {}) {
       return this.client().issueLabels(variables);
     },
+    async listComments(variables = {}) {
+      return this.client().comments(variables);
+    },
+    fnObjects() {
+      return {
+        [constants.RESOURCE_TYPE.COMMENT]: this.listComments,
+        [constants.RESOURCE_TYPE.ISSUE]: this.listIssues,
+        [constants.RESOURCE_TYPE.ISSUE_LABEL]: this.listIssueLabels,
+        [constants.RESOURCE_TYPE.USER]: this.listUsers,
+        [constants.RESOURCE_TYPE.PROJECT]: this.listProjects,
+        [constants.RESOURCE_TYPE.TEAM]: this.listTeams,
+      };
```

Comment:
This is a good way to map the functions!

Optional: If there is a way to "automatically" map the functions without having to manually add a line for each `resourceType`, that would be great.
For example, suppose we have to add `RESOURCE_TYPE.MILESTONE`, then we would have to add the code for the mapping to `this.listMilestones`. Would be great if there is a way to avoid this.

Reply:
Yes. There is a way.